### PR TITLE
Various website tweaks

### PR DESCRIPTION
**Pull request by graymalkin**

### CODE REVIEW

File: content/resources/radio.md [hunk 1]
```diff
@@ -5,55 +5,71 @@ menu:
...
-# History
+</style>
+{{% picture "radio/FT991.jpg" %}}
 
-We did have a Amateur Radio Society club which was run by the students which was
-very successful through the 70s and 80s up to about 1995.
 
-It's popularity declined after 1995 and then it was very difficult to get the 15
-students to sign up, that the student union required to form the club.
+We also run a radio workshop every few weekends, allowing members to work on obtaining their amateur radio licenses, and try and make contact with people across Europe with their callsigns! We currently have more than 10 members who have passed their Foundation exams and more than 5 have Intermediate Licenses.
```
Comment (egelmex) [inline content/resources/radio.md]: Not sure this is correct...

File: content/resources/radio.md [hunk 2]
```diff
@@ -5,55 +5,71 @@ menu:
     main:
         parent: resources
 ---
+<style>
+.sstv img {
+	width: 32%;
+}
 
-# History
```
Comment (egelmex) [inline content/resources/radio.md]: Would it not be good to have a history section? Though I agree replacing this copy would be a good idea.

File: content/resources/radio.md [hunk 3]
```diff
@@ -5,55 +5,71 @@ menu:
...
+PSK31 is a data mode for sending text rather than voice, it is similar to morse, yet it is faster, designed for cases when there’s low SNR (Signal-to-noise ratio). PSK31 can usually be decoded pretty successfully even when there’s lots of background noise and the signal is barely noticeable. Due to this, it is often used on relatively low powers since it can still go long distances and be heard. We have made contact with people from the Dolomites (NE Italy) using 20W. 
+
+# Licenses
+If you are interested in amateur radio, there are 3 licenses available in the UK, each builds on the last.
 
 ### Foundation License (exam cost: £27.50)
  - Allowed up to 10 watts of radio output
  - May not build own equipment
  - May build equipment from approved kits
+ - <a href="http://rsgb.org/main/clubs-training/for-students/foundation/"> More information </a>
```
Comment (egelmex) [inline content/resources/radio.md]: Would it be better to use markdown syntax for links?

File: themes/TinkerSoc/layouts/partials/footer.html [hunk 4]
```diff
@@ -26,5 +26,15 @@
     </div>
 </footer>
 
+<!-- Google Analytics -->
```
Comment (egelmex) [inline themes/TinkerSoc/layouts/partials/footer.html:29]: Is this our existing analytics or a new account?

File: themes/TinkerSoc/layouts/partials/footer.html [hunk 5]
```diff
@@ -26,5 +26,15 @@
     </div>
 </footer>
 
+<!-- Google Analytics -->
+<script>
```
Comment (egelmex) [inline themes/TinkerSoc/layouts/partials/footer.html:30]: I think hugo has a short code for this

File: content/resources/radio.md (hunk 1 above)
Comment (graymalkin) [reply content/resources/radio.md]: This was briefly correct. I've had this PR sat around for a while.

File: content/resources/radio.md (hunk 3 above)
Comment (graymalkin) [reply content/resources/radio.md]: It would.

File: themes/TinkerSoc/layouts/partials/footer.html (hunk 4 above)
Comment (graymalkin) [reply themes/TinkerSoc/layouts/partials/footer.html:29]: Yeah, this is pulled straight out of the previous site.

File: themes/TinkerSoc/layouts/partials/footer.html (hunk 5 above)
Comment (graymalkin) [reply themes/TinkerSoc/layouts/partials/footer.html:30]: There is but it doesn't get evaluated in partials, so this method is cleaner.

File: themes/TinkerSoc/static/css/style.css [hunk 6]
```diff
@@ -337,7 +337,7 @@ figure img {
 
 .video_shade {
     background-color: rgba(0,0,0,0.4);
-    background-image: url('/tinker-new/images/shade.svg');
+    background-image: url('/images/shade.svg');
```
Comment (graymalkin) [inline themes/TinkerSoc/static/css/style.css]: This is quite ugly.

File: content/resources/radio.md (hunk 1 above)
Comment (egelmex) [reply content/resources/radio.md]: HaHa fair enough, suppose it's not too bad if they plan to run something this year 👍

**Comment by graymalkin** [comment]
Depends on what the committee want to do. It's their show! :beers:

File: themes/TinkerSoc/static/css/style.css (hunk 6 above)
Comment (egelmex) [reply themes/TinkerSoc/static/css/style.css]: As in looks ugly? or bad implementation?

File: content/resources/radio.md (hunk 1 above)
Comment (graymalkin) [reply content/resources/radio.md]: Depends on what the committee want to do. It's their show! ￼:beers:

File: themes/TinkerSoc/static/css/style.css (hunk 6 above)
Comment (graymalkin) [reply themes/TinkerSoc/static/css/style.css]: Looks ugly. See http://staging.tinkersoc.org/

Comment (graymalkin) [reply themes/TinkerSoc/static/css/style.css]: I've fixed this in 2001b8b

File: content/resources/radio.md (hunk 3 above)
Comment (graymalkin) [reply content/resources/radio.md]: Addressed with b741a78.

**Review by egelmex — APPROVED** [review]

**Pull request merged**